snippets/System.Text/Rune/csharp/EncodeRune.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RuneSamples
+{
+    public static class EncodeRune
+    {
+        public static void Run()
+        {
+            Console.WriteLine("Converting ox emoji to UTF-16 and UTF-8");
+            var rune = new Rune("🐂".ToCharArray()[0], "🐂".ToCharArray()[1]);
```

Comment:
I might suggest the following as easier to understand over deconstructing the string into its constituent chars and feeding them to the ctor:

```cs
Rune rune = Rune.GetRuneAt("🐂", 0);
```